recipe/meta.yaml

```
@@ -25,20 +28,24 @@ requirements:
   run:
     - python
     - coloredlogs
+    - datasets
     - sympy
     - pytorch >=1.9
-    - transformers >=4.20.1
     - huggingface_hub >=0.8.0
+    - transformers >=4.26.0
+    # transformers[sentencepiece] contains sentencepiece and protobuf, see https://github.com/huggingface/transformers/blob/v4.32.1/setup.py#L299
+    - sentencepiece >=0.1.91,!=0.1.92
+    - protobuf
     - packaging
     - numpy
```

```suggestion
    - numpy
  run_contrained:
    - onnxruntime >=1.11.0
    - datasets >=1.2.1
    - protobuf>=3.20.1
```
because of https://github.com/huggingface/optimum/blob/v1.12.0/setup.py#L45